staging/src/k8s.io/apiserver/pkg/endpoints/handlers/watch.go

```diff
@@ -202,6 +202,19 @@ type WatchServer struct {
 	metricsScope string
 }
 
+// byteCounter is a wrapper around io.Writer that counts bytes written.
```

Comment:
Would be good to document assumptions about thread safely and writing being synchronous.